ax/modelbridge/modelbridge_utils.py

```diff
@@ -1061,7 +1061,8 @@ def observation_data_to_array(
 
 
 def observation_features_to_array(
-    parameters: List[str], obsf: List[ObservationFeatures]
+    parameters: List[str],
+    obsf: Union[List[ObservationFeatures], List[List[ObservationFeatures]]],
```

Comment:
This will raise an error `'list' object has no attribute 'parameters'` if obsf is `List[List[ObservationFeatures]]`. Let's revert to `List[ObservationFeatures]` only since that is the only active use case.